src/harness/fakesHosts.ts

```diff
@@ -335,7 +337,7 @@ export class CompilerHost implements ts.CompilerHost {
         return vpath.resolve(this.getDefaultLibLocation(), ts.getDefaultLibFileName(options));
     }
 
-    public getSourceFile(fileName: string, languageVersion: number): ts.SourceFile | undefined {
+    public getSourceFile(fileName: string, languageVersionOrOptions: ts.ScriptTarget | ts.CreateSourceFileOptions): ts.SourceFile | undefined {

```

Comment:
I have absolutely no idea how this was compiling before.

Reply:
This is no longer needed in the PR but still technically correct; I'll pull it out for a different PR.